os/board/rtl8721csm/src/component/os/tizenrt/rtk_netmgr.c

```diff
@@ -437,6 +509,25 @@ trwifi_result_e wifi_netmgr_utils_connect_ap(struct netdev *dev, trwifi_ap_confi
 		}
 	}
 
+	ap_channel = 0xffff;

```

Comment:
how about to pass it as function parameter. the use of global variable should be minimized.

Reply:
@pillip8282  Modified as suggested, please help to review.

Reply:
you misunderstood my point. I think make ap_channel to a local variable. and pass it to cmd_wifi_connect() like this.
```
ret = cmd_wifi_connect(ap_connect_config, arg, ap_channel);
```

Reply:
@pillip8282 , Modified the API as per suggestion and made ap_channel as local variable.